src/main/python/ttconv/scc/mid_row_codes.py

```diff
@@ -56,14 +60,53 @@ def get_name(self) -> str:
     """Retrieves SCC Mid-Row Code name"""
     return self.name
 
+  def get_color(self) -> typing.Optional[NamedColors]:
+    """Returns SCC Mid-Row Code color"""
+    style_bits = self._channel_1 & 0x000F
+
+    if style_bits in [0x00, 0x01]:
+      return NamedColors.white
+    if style_bits in [0x02, 0x03]:
+      return NamedColors.green
+    if style_bits in [0x04, 0x05]:
```

Comment:
I would use tuples, e.g. `(0x04, 0x05)`, instead of lists since tuples are immutable.

Reply:
Using tuples would allow the use a dictionary instead of `if...return` statements, e.g.

```
mapping = {
(0x00, 0x01) : NamedColors.white
}
```

Reply:
Done in 681cd3a5e52267ab5ca24850bde5bc52d975cbc6